src/components/TableDialog.vue

```diff
@@ -218,6 +219,11 @@ export default defineComponent({
       }));
     });
 
+    const edgeTable = computed(() => {
+      const keys = Object.keys(sampleRows.value[0] || {});
+      return keys.includes('_from') && keys.includes('_to');
```

Comment:
There's a better way to do this. Extract the object from `sampleRows` as you are doing in line 223, then call `.hasOwnProperty()` on it to detect the `_from` and `_to` attributes.

You may run into a linter error when you do this--this [documentation](https://eslint.org/docs/rules/no-prototype-builtins) explains what's going on and how to fix it.

Reply:
That's helpful, thank you!

See 701bec8